_includes/head.html

```diff
@@ -2,11 +2,9 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
 	
-	<title>Engo</title>
 	<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/foundation/6.2.0/foundation.min.css">
 	<style type="text/css">
 		body { text-align: center; }
 		.row { max-width: 35rem; }
 		footer p { font-size: small; color: gray; }
 	</style>
-</head>
```

Comment:
I don't think we should break the markup here.

[A better solution would be to use Jekyll variables](https://jekyllrb.com/docs/variables/)